docs/source/index.rst

```diff
@@ -58,3 +58,10 @@ architectures, and common image transformations for computer vision.
    TorchElastic <https://pytorch.org/elastic/>
    TorchServe <https://pytorch.org/serve>
    PyTorch on XLA Devices <http://pytorch.org/xla/>
+
+
+Indices and tables
```

Comment:
As you noted, this is empty. Why is this expected to be empty? If it's empty, should we just remove it?

Reply:
Well search page is empty as that's what sphinx does.
We can either keep it like [pytorch lightning](https://pytorch-lightning.readthedocs.io/en/latest/#indices-and-tables) does or simply avoid it and keep only Index.

Reply:
Ok, the index might be useful.

I can't see the rendered docs but when I tried locally last time, it wasn't generated though. Is this normal?

Reply:
Let me update the PR accordingly, maybe the coverage is causing error. I could build index locally though.